alert-object.libsonnet

```diff
@@ -0,0 +1,31 @@
+local mi = import 'mixin.libsonnet';
```

Comment:
can you move this file to extras folder?

Reply:
will do.